ui_framework/dist/ui_framework.css

```diff
@@ -1915,6 +1915,40 @@ body {
     /* 1 */ }
 
 /**
+ * 1. When it expands, the search input should lie on top of the content.
+ * 2. Expand as the user enters text.
+ * 3. Cap the height so that we can offer auto-completion suggestions below the input.
+ * 4. When the input is collapsed, we want text input to be truncated with an ellipsis.
+ */
+.kuiLocalSearchInput--expandable {
```

Comment:
i see quite some duplication between this and `_local_search.scss` ... whats the idea behind this ?